definitions-terms.md

```diff
@@ -0,0 +1,108 @@
+# Definitions and Terms
+
+A collection of definitions and terms used within this repository.
+
+## TOC
+
+* [Fingerprint](#fingerprint)
+* [Fully Qualified Artifact Reference](#registry-fully-qualified-reference)
+* [Multi-Tenant Registry](#multi-tenant-registry)
+* [OCI Annotations](#oci-annotations)
+* [OCI Descriptor](#oci-descriptor)
+* [OCI Index](#oci-index)
+* [OCI Manifest](#oci-manifest)
+* [Registry](#registry)
+* [SBoM](#sbom)
+* [Subject](#subject)
+* [Tag](#tag)
+
+## Artifact / Image
+
+Artifacts are a generalization of how images are stored in an instance of the [OCI Distribution Spec][oci-distribution-spec]. An [OCI Image][oci-image] is a type of [OCI Artifact][oci-artifact]. When referencing artifacts, within a [fully qualified reference](#registry-fully-qualified-reference), the artifact is right most element of the namespace, combined with the `:tag`.  
+Reference: [OCI Artifacts][oci-artifact]  
+![](./media/artifact-ref.png)  
+
+
+## Fingerprint
+
+A short identifier of a given public key.  
+Reference: [Fingerprint][fingerprint]
+
+## Multi-Tenant Registry
+
+A collection of unique [registries](#registry), stored within an instance of an [OCI Distribution Spec][oci-distribution-spec]. Registry operators use one of two methods to uniquely identify an org. In either case, information within that org is considered unique to the org, including all key references and authentication.
+
+## OCI Annotations
+
+A key-value map that can be associated with [OCI Descriptors][oci-descriptor] and [OCI Image manifests][oci-manifest].  
+OCI spec reference: [OCI Annotations][oci-annotations]
+
+## OCI Descriptor
+
+A structure describing content, including the media type, a content-addressable digest, the size, and other properties. Descriptors are used to describe layers and configuration in a manifest.  
+OCI spec reference: [OCI Descriptor][oci-descriptor]
+
+## OCI Index
+
+A higher-level collection of [image manifests][oci-manifest], or other [oci indexes][oci-index], typically used to describe platform-specific (architecture and operating-system) images that can be identified collectively and referred to together. The specific image manifests are identified by modified [descriptors][oci-descriptor] with additional properties and restrictions. [OCI Artifacts][oci-artifact] takes this a step further and provides for an OCI Index to reference a collection of artifacts, such as an image, a deployment chart, its SBoM and deployment configuration references.  
```

Comment:
> [OCI Artifacts][oci-artifact] takes this a step further and provides for an OCI Index to reference a collection of artifacts, such as an image, a deployment chart, its SBoM and deployment configuration references.

Suggestion: [OCI Artifacts][oci-artifact] are another use-case for indexes where an OCI Index can reference a collection of artifacts, such as an image, a deployment chart, its SBoM and deployment configuration references.